helix-core/src/test/java/org/apache/helix/integration/multizk/MultiZkTestBase.java

```diff
@@ -0,0 +1,150 @@
+package org.apache.helix.integration.multizk;
```

Comment:
Copied Before and After class from pre-existing tests in MultiZkConnectionConfig to avoid code repetition for the other test cases.

Reply:
please add apache license, thanks